<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server_Telemetry_Info_Form</class>
 <widget class="QDialog" name="Server_Telemetry_Info_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>183</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>183</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>单一遥测信息</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lab_addr">
         <property name="text">
          <string>遥测地址</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_coeff">
         <property name="text">
          <string>系数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_clientID">
         <property name="text">
          <string>对应客户端ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_clientTA">
         <property name="text">
          <string>对应客户端遥测地址</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_describe">
         <property name="text">
          <string>描述</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_addr"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_coeff"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_clientID"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_clientTA"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_describe"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
